tests/test_halo.py

```diff
@@ -71,6 +79,29 @@ def test_basic_spinner(self):
         self.assertEqual(output[1], '{0} foo'.format(frames[1]))
         self.assertEqual(output[2], '{0} foo'.format(frames[2]))
 
+    def test_spinner_getter(self):
+        instance = Halo()
+        if _utils.is_supported():
+            default_spinner_value = "dots"
+        else:
+            default_spinner_value = "line"
+
+        instance.spinner = default_spinner_value
+        self.assertEqual(default_spinner, instance.spinner)
+
+        instance.spinner = "This_spinner_do_not_exist"
+        self.assertEqual(default_spinner, instance.spinner)
+
+        instance.spinner = -123
+        self.assertEqual(default_spinner, instance.spinner)
+
+    def test_windows_users_get_their_default_spinner(self):
```

Comment:
We shouldn't add OS specific tests. We run the code on Travis(Linux) and Appveyor(Windows) for testing in both operating systems.